src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_ExcelCompat_PowerFxV1Compat.txt

```diff
@@ -1032,4 +1032,179 @@ true
 "Thursday, January 1, 1970 12:00:00 AM"
 
 >> Text(Date(1970, 1, 1), DateTimeFormat.LongDateTime24)
-"Thursday, January 1, 1970 0:00:00"
\ No newline at end of file
+"Thursday, January 1, 1970 0:00:00"
+
+// Block exponential notation with decimals, escaping character and scaling modifier
+>> Text(1234567.1234567, "0.00E+0.0")
+Errors: Error 0-34: The function 'Text' has some invalid arguments.|Warning 22-33: Incorrect format specifier for 'Text'.
```

Comment:
Those are all compile-time errors, but it's possible that the compiler won't catch this issue - for example, if we use an expression like the one below. Can you add some of those tests using that as well? No need to do all of them, since the code is shared, but it would be good to get a few (~25%) to ensure coverage on that path as well.

    >> With({fmt:"0.00E+0.0"},Text(1234567.1234567,fmt))

Reply:
Added these tests. thanks